### DISCUSSION TITLE
Problem when testing Select component

### DISCUSSION THREAD

**Discussion by kel666** [ant-design/ant-design#29062] [General]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/cannot-test-select-component-85vv0)

### Steps to reproduce
Run the test

### What is expected?
Executing the 
`fireEvent.mouseDown(combo); 
`doesn't trigger rendering error

### What is actually happening?
This error appear and event is not triggered:

```
Warning: An update to List inside a test was not wrapped in act(...).
When testing, code that causes React state updates should be wrapped into act(...):

```
In fact, I'm already using act()

| Environment | Info |
|---|---|
| antd | 4.11.2 |
| React | 17.0.1 |
| System | Win 7 |
| Browser | Chrome 87 |

**Comment by pgsill** [comment]
@kel666 Not sure how correct this is, but I managed to get rid of it by doing this:

```
await act(async () => {
    await waitFor(() => fireEvent.click(dropdownOption));
});
```
